site/content/docs/5.3/migration.md

```diff
@@ -22,6 +22,9 @@ If you need more details about the changes, please refer to the [v5.3.1 release]
 - **Nav and tabs**
   - <span class="badge bg-success">New</span> Added support for JavaScript Tabs' <kbd>Home</kbd> and <kbd>End</kbd> keys.
 
+- **Side navigation**
```

Comment:
Don't forget to move this part to v5.3.x when ready 😉

Reply:
I let it there for now since I don't know when this component will be merged. I'll move it once I know it!